### PR TITLE
Added staging db

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1217

### Change description ###
Creating database to be used for functional tests by staging pod.
• This PR is dependent on hmcts/bulk-scan-shared-infrastructure#171 (app gw changes)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by apereverzin** [comment]
@mdayican Yes, updated PR description

**Comment by mdayican** [comment]
> @mdayican Yes, updated PR description

actually this can go , this is just creating. to start using you need to wait.

File: infrastructure/main.tf [hunk 1]
```diff
@@ -40,6 +40,21 @@ module "bulk-scan-db" {
...
 
+# Staging DB to be used by AAT staging pod for functional tests
+module "bulk-scan-staging-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = "${var.product}-${var.component}-staging"
+  location           = "${var.location_db}"
+  env                = "${var.env}"
+  database_name      = "bulk_scan"
+  postgresql_user    = "bulk_scanner"
+  postgresql_version = "10"
```
Comment (mdayican) [inline infrastructure/main.tf:51]: this shoul be new version . v 11

**Comment by apereverzin** [comment]
@mdayican Anyway it is better to have infrastructure changes in place first

**Pull request merged**